src/Language/Haskell/Interpreter.hs

```diff
@@ -36,7 +36,7 @@ module Language.Haskell.Interpreter(
     -- ** Type inference
      typeOf, typeChecks, kindOf,
     -- ** Evaluation
-     interpret, as, infer, eval,
+     interpret, unsafeInterpret, as, infer, eval,
```

Comment:
Probably a good idea to export this as part of `Language.Haskell.Interpreter.Unsafe` instead.